include/oneapi/dpl/internal/dynamic_selection_impl/dynamic_load_policy.h

```diff
@@ -0,0 +1,170 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) 2023 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef _ONEDPL_DYNAMIC_LOAD_POLICY_H
+#define _ONEDPL_DYNAMIC_LOAD_POLICY_H
+
+#include <atomic>
+#include <memory>
+#include <limits>
+#include <mutex>
+#include <vector>
+#include <exception>
+#include <type_traits>
+#include <utility>
+#include "oneapi/dpl/internal/dynamic_selection_traits.h"
+#if _DS_BACKEND_SYCL != 0
+    #include "oneapi/dpl/internal/dynamic_selection_impl/sycl_backend.h"
+#endif
+
+namespace oneapi {
+namespace dpl{
+namespace experimental{
+
+#if _DS_BACKEND_SYCL != 0
+  template <typename Backend=sycl_backend>
+#else
+  template <typename Backend>
+#endif
+  struct dynamic_load_policy {
+    private:
+    using backend_t = Backend;
+    using load_t = int;
+    using execution_resource_t = typename backend_t::execution_resource_t;
+
+    public:
+    //Policy Traits
+    using resource_type = typename backend_t::resource_type;
+    using wait_type = typename backend_t::wait_type;
+
+    //private:
+    struct resource_t {
+      execution_resource_t e_;
+      std::atomic<load_t> load_;
+      resource_t(execution_resource_t e) : e_(e) {load_=0;}
```

Comment:
As far as I know atomic initialization is better to do in initializer list but not in constructor body, because `atomic` assignment is not required here:
- https://en.cppreference.com/w/cpp/atomic/atomic/operator%3D
- https://en.cppreference.com/w/cpp/atomic/atomic/atomic

Reply:
Fixed.